ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -47,6 +47,7 @@
                   predefined-parameters: |
                     BRANCH=${GIT_BRANCH}
                     FORCE=True
+                    FLAVOR=notcmalloc
```

Comment:
I think these `FLAVOR` additions just got merged with a13e43401dff61ec5d0e705d76bde70ead72a900 right?

Reply:
@alfredodeza yes. the github's merge button takes this into consideration. so this PR is now equivalent to bd61b62 + a8a926d.

Reply:
okay, rebased and repushed.